runtime/autoload/tutor.vim

```diff
@@ -100,18 +100,18 @@ function! tutor#ApplyMarksOnChanged()
     endif
 endfunction
 
-function! tutor#CheckLine(line)
+function! tutor#CheckLine(lnum)
```

Comment:
In all 3 calls to this method the given param is given as lnum. Thus it makes only sense to also treat it as lnum here. Line would indicate a different value.